### PR TITLE
[WFLY-17332] Upgrade Narayana to 6.0.0.CR1

### PR DESCRIPTION
**Pull request by jmfinelli**
This PR addresses [WFLY-17332](https://issues.redhat.com/browse/WFLY-17332)

### CODE REVIEW

**Comment by jmfinelli** [comment]
For some minor "technical" problems, Narayana has to be released again. Next version will be CR2 (which will be only a name change from CR1 to CR2). Once CR2 is available, this PR will be update and the `hold` label removed.

**Comment by jmfinelli** [comment]
> For some minor "technical" problems, Narayana has to be released again. Next version will be CR2 (which will be only a name change from CR1 to CR2). Once CR2 is available, this PR will be updated and the `hold` label removed.

Little change of program. The Narayana team is happy to go ahead with CR1 so this PR is ready to be reviewed! Thanks

File: boms/standard-ee/pom.xml [hunk 1]
```diff
@@ -2495,6 +2495,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.narayana.xts</groupId>
+                <artifactId>byteman_support</artifactId>
```
Comment (bstansberry) [inline boms/standard-ee/pom.xml:2500]: I this used in production code? AFAICT it is not.

If not please remove this and restore the section of the root pom that touches it and @parsharma will handle this dependency as part of #16471 / https://issues.redhat.com/browse/WFLY-17412.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Other than the byteman_support topic, this LGTM.

**Comment by bstansberry** [comment]
Also, when is 6.0.0.Final expected?

**Comment by jmfinelli** [comment]
> Also, when is 6.0.0.Final expected?

6.0.0.Final should be release shortly after merging this PR :-) In fact, we're very confident that 6.0.0.CR1 is ready to be released as `Final` but we need to get back our CI to confirm our opinion ;-)

File: boms/standard-ee/pom.xml (hunk 1 above)
Comment (jmfinelli) [reply boms/standard-ee/pom.xml]: Sorted! Thanks Brian!

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @jmfinelli!

**Pull request merged**